src/Visualizer/package.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
we already have package.json in root folder. can you please merge packages needed into the root file?

Reply:
This is because they are react-related libraries used only by Visualizer folder.

Reply:
Can you please add a comment here, about explaining multiple package.json structure?

Reply:
`package.json` used here are only packages for build react, and we divided the packages because there are dependencies that we need to create visualizers(build by react) but do not need to operate extensions.

Reply:
Not related with `react` but in general. I just want to leave some histroy about this PR with multiple package.json files.
This is the first PR, as of now, with another package.json.

Reply:
I didn't understand exactly what you mean.
Are you saying that you need a reason to use multiple package.json files?

Reply:
> `package.json` used here are only packages for build react, and we divided the packages because there are dependencies that we need to create visualizers(build by react) but do not need to operate extensions.

Do you mean the `dependencies` are used at development time only, and not used at runtime?
Then, can they be put in `devDependencies`?

Reply:
> you need a reason to use multiple package.json files?

No, I'd like to leave a comment here about multiple package,.json files. I googled but failed (I'm not used to this domain) to get information about it.

Reply:
What you mean is just transferring the content of the `dependencies` to the `devDependencies`?
If so, there is no problem and it is possible.

Reply:
> No, I'd like to leave a comment here about multiple package,.json files. I googled but failed (I'm not used to this domain) to get information about it.

The app inside the `visualizer folder` recognizes `visualizer folder` as root.
package.json is usually placed in the root directory in a file structure.
If you look at this [link](https://github.com/yarnpkg/yarn/issues/6160), it refers to package management for multi-routes.

Reply:
> What you mean is just transferring the content of the `dependencies` to the `devDependencies`?
> If so, there is no problem and it is possible.

No, just your answer is okay. For your information, I am interested in what is the part of license check and at this moment, checking the main license for each modules seems enough.